gnome-extra/nautilus-dropbox/nautilus-dropbox-2019.02.14.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```

Comment:
7 should be available here.

Reply:
Eclass `gnome2` does not support EAPI=7 :-(

Reply:
...which is solved by dropping it :p